repeatedDNASequences.py

```diff
@@ -0,0 +1,15 @@
+class Solution(object):
+    def findRepeatedDnaSequences(self, s):
+        """
+        :type s: str
+        :rtype: List[str]
+        """
+        res = []
+        if len(s) < 10:
+            return res
+        summary = {}
+        for i in range(len(s) - 9):
+            if s[i:i+ 10] in summary and summary[s[i:i+10]] == 1:
```

Comment:
You can define `s[i : i + 10]` as a variable inside the for loop given that it is used 5 times in the for loop.

Reply:
Sure